src/unit_test_stubs/share_esmf/PrigentRoughnessStreamType.F90

```diff
@@ -0,0 +1,119 @@
+module PrigentRoughnessStreamType
+
+  !-----------------------------------------------------------------------
```

Comment:
Add a note here and to the Paleo issue that this can be removed and the actual one used when it's allowed for Prigent streams to be off. This would save on code duplication and minor differences in how it's done for testing.